sites/skeleton.dev/src/content/docs/components/popover/svelte.mdx

```diff
@@ -0,0 +1,97 @@
+---
+layout: '@layouts/LayoutDoc.astro'
+---
+
+export const components = componentSet;
+
+import Default from '@examples/components/popover/svelte/default.svelte';
+import DefaultRaw from '@examples/components/popover/svelte/default.svelte?raw';
+
+import Programmatic from '@examples/components/popover/svelte/programmatic.svelte';
+import ProgrammaticRaw from '@examples/components/popover/svelte/programmatic.svelte?raw';
+
+import Arrow from '@examples/components/popover/svelte/arrow.svelte';
+import ArrowRaw from '@examples/components/popover/svelte/arrow.svelte?raw';
+
+import ZIndex from '@examples/components/popover/svelte/z-index.svelte';
+import ZIndexRaw from '@examples/components/popover/svelte/z-index.svelte?raw';
+
+import Styling from '@examples/components/popover/svelte/styling.svelte';
+import StylingRaw from '@examples/components/popover/svelte/styling.svelte?raw';
+
+<Preview client:load>
+	<Fragment slot="preview">
+		<Default client:visible />
+	</Fragment>
+	<Fragment slot="code">
+		<Code code={DefaultRaw} lang="svelte" />
+	</Fragment>
+</Preview>
+
+## Arrow
+
+You may optionally enable arrows via the `Arrow` and `ArrowTip` component parts. Note that Zag.js opts to style these with CSS custom properties, which can be adjusted using a `style` attribute.
+
+<Preview client:load>
+	<Fragment slot="preview">
+		<Arrow client:visible />
+	</Fragment>
+	<Fragment slot="code">
+		<Code code={ArrowRaw} lang="svelte" />
+	</Fragment>
+</Preview>
+
+## Z-Index
+
+By default we do not take an opinionated stance regarding z-index stacking. The result is the component can sometimes be occluded beneath other elements with a higher index. The Z-Index can controlled by applying a utility class to the `Positioner` component part.
```

Comment:
@Hugos68 the example was updated but not the docs. So I've modified this. Including a warning about needing `!`. I've updated both Svelte and React docs here - but you may need to carry this over to the Tooltip, Modal, and Combobox docs as well.